remo/domain/dataset.py

```diff
@@ -86,6 +87,8 @@ def add_data(
             class_encoding: specifies how to convert labels in annotation files to readable labels. If None,  Remo will try to interpret the encoding automatically - which for standard words, means they will be read as they are. 
                 See also: :class:`remo.class_encodings`.
 
+            wait_for_complete: blocks function until upload data completes
```

Comment:
in case if user want to wait until data upload completes, then True
otherwise - False

Reply:
@vmanilo ok - asking why you think the user might want to wait or why he could want to not wait. I'm asking because unless we have a specific reason, i guess is better to just simplify the function and remove this option

Reply:
in some cases you want to run the job (like uploading big dataset) and continue work with SDK, but if the function blocks the call and waits for completion you won't be able continue your work